### ISSUE TITLE
[REVIEW]: ShinySSD: Species Sensitivity Distributions for Ecotoxicological Risk Assessment

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#785]
**Submitting author:** @flor14 (<a href="http://orcid.org/0000-0002-0041-097X">María Florencia D&#39;Andrea</a>)
**Repository:** <a href="https://github.com/flor14/paper" target ="_blank">https://github.com/flor14/paper</a>
**Version:** v1.0.0
**Editor:** @karthik
**Reviewer:** @kylehamilton
**Archive:** <a href="https://doi.org/10.5281/zenodo.3233033" target="_blank">10.5281/zenodo.3233033</a>

## Status

[![status](http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3/status.svg)](http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3)

Status badge code:

```
HTML: <a href="http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3"><img src="http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3/status.svg"></a>
Markdown: [![status](http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3/status.svg)](http://joss.theoj.org/papers/55eab2244e9d856826450cb8d9416ec3)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/flor14/paper" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@kylehamilton, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.theoj.org/about#reviewer_guidelines. Any questions/concerns please let @karthik know.

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/flor14/paper">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Version:** Does the release version given match the GitHub release (v1.0.0)?
- [x] **Authorship:** Has the submitting author (@flor14) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks. @kylehamilton it looks like you're currently assigned as the reviewer for this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #785 with the following error: 

 /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/author.rb:36:in `block in build_affiliation_string': Problem with affiliations for María Florencia D’Andrea, perhaps the affiliations index need quoting? (RuntimeError)
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/author.rb:35:in `each'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/author.rb:35:in `build_affiliation_string'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/author.rb:11:in `initialize'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon.rb:96:in `new'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon.rb:96:in `block in parse_authors'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon.rb:94:in `each'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon.rb:94:in `parse_authors'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon.rb:70:in `initialize'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/processor.rb:32:in `new'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/lib/whedon/processor.rb:32:in `set_paper'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/bin/whedon:37:in `prepare'
	from /app/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	from /app/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	from /app/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	from /app/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	from /app/vendor/bundle/ruby/2.3.0/bundler/gems/whedon-195e6d124d0f/bin/whedon:99:in `<top (required)>'
	from /app/vendor/bundle/ruby/2.3.0/bin/whedon:22:in `load'
	from /app/vendor/bundle/ruby/2.3.0/bin/whedon:22:in `<main>'

**Comment by flor14** [comment]
If there is something unclear in the submission I can improve it

**Comment by arfon** [comment]
@flor14 the multiple affiliations need quoting. i.e. change this:

```
affiliation: 1,2
```

to this in your `paper.md`

```
affiliation: "1,2"
```

**Comment by flor14** [comment]
done!

**Comment by arfon** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by flor14** [comment]
Great.

Only one thing:
The citation that says "Plan protection Products" is not correct, I have
changed the bib file in order to modify that

Sorry about that and thank you

**Comment by karthik** [comment]
@kylehamilton Just wanted to check in on the status of your review. Are you waiting on @flor14 for any pending items? 🙏

**Comment by flor14** [comment]
I have not pending items yet

**Comment by kylehamilton** [comment]
Sorry for the delay I'll have this completed by the end of the day.

**Comment by karthik** [comment]
Thanks @kylehamilton Let me know if you need any help. 🙏

**Comment by flor14** [comment]
I can help if something is confusing.

**Comment by flor14** [comment]
Hello! I was thinking about this today. Is there something can I start improving? Thank you so much

**Comment by karthik** [comment]
@kylehamilton Gentle ping to see if you are able to wrap your review soon?

**Comment by karthik** [comment]
Hi @kylehamilton, checking in again to see if you've had time to wrap this review up.
 🙏

**Comment by kylehamilton** [comment]
Sorry @karthik I posted the things that needed to be addressed in the issues of the repo https://github.com/flor14/paper/issues 

1. Make the software into an R package so everything would be contained and centralized https://github.com/flor14/paper/issues/2
2. Add tests https://github.com/flor14/paper/issues/3
3. Just want to make sure that there is a DOI for the source code I didn't see this in the .bib file or in the readme https://github.com/flor14/paper/issues/4
4. A minor change would in the name of the repo for production code should be changed from paper to ShinySSD 
5. Needs version numbers, shifting this to an R package would help fix this https://github.com/flor14/paper/issues/6
6. I did not see community guidelines https://github.com/flor14/paper/issues/7 

If I did this wrong then I'm really sorry I thought that if I posted them as issues in the repo then it would move on from there.

**Comment by flor14** [comment]
Thanks! I will do it soon

**Comment by flor14** [comment]
I have completed the review.
I change a little bit the README.md, it is different than the paper.md for publication.

Thank you

**Comment by karthik** [comment]
Thank you @flor14 
@kylehamilton Could you take a look when you get a chance?

🙏

**Comment by karthik** [comment]
@flor14 I noticed there are couple more issues to fix before we can proceed with this. Can you please address the last two issues (5/6) that Kyle raised. I'd like to wrap this up before the end of the year. Thanks.

**Comment by flor14** [comment]
I will have a look a soon as possible. Thank you.

**Comment by flor14** [comment]
Hello. The last 4 months I have been finishing my PhD so I had no time to
check this (and I will not have until the last week of February). I am not
sure how is JOSS policy about timing, but I can test everything again and
make improvements on February. Sorry about this, it took a bit longer than
expected.
Regards

**Comment by labarba** [comment]
Hi @flor14 — Sure. Take your time, but ping us back here when you are ready for the review to continue.

**Comment by flor14** [comment]
Well, I finished improving the app, I hope I have interpreted the comments
correctly. Everything seems to work well.
Thanks for the valuable suggestions.

**Comment by labarba** [comment]
@karthik — take a look at this one to continue the review 🙏

**Comment by karthik** [comment]
The updates look good to me. @kylehamilton Could you have a look please and check off your remaining items (or raise more issues?) 🙏

**Comment by labarba** [comment]
Hi @kylehamilton 👋 — It looks like we're waiting for you to check the author's revision and responses to your comments. Can you give us a status update?

**Comment by flor14** [comment]
Hello. Maybe he is not receiving the notifications? Is not possible to send
him an email?

**Comment by karthik** [comment]
I'll send Kyle an email shortly.

**Comment by labarba** [comment]
👋 @karthik — did you ever get a reply from @kylehamilton ? 
This looks stalled and may need more active nudging. Thanks1

**Comment by karthik** [comment]
Thanks Lorena, I'll follow up again. Otherwise try to assign someone new.

**Comment by karthik** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by karthik** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00785/joss.00785/10.21105.joss.00785.pdf)

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- None

MISSING DOIs

- https://doi.org/10.1016/j.ecoenv.2016.08.023 may be missing for title: Aquatic toxicity structure-activity relationships for the zwitterionic surfactant alkyl dimethyl amine oxide to several aquatic species and a resulting species sensitivity distribution
- https://doi.org/10.2903/j.efsa.2013.3290 may be missing for title: Guidance on tiered risk assessment for plant protection products for aquatic organisms in edge-of-field surface waters
- https://doi.org/10.18637/jss.v064.i04 may be missing for title: fitdistrplus: An R Package for Fitting Distributions
- https://doi.org/10.1002/etc.5620190233 may be missing for title: Applying species-sensitivity distributions in ecological risk assessment: assumptions of distribution type and sufficient numbers of species
- https://doi.org/10.1201/9781420032314 may be missing for title: Species sensitivity distributions in ecotoxicology
- https://doi.org/10.1016/s0025-326x(01)00327-7 may be missing for title: Species sensitivity distributions: data and model choice
- https://doi.org/10.1002/9780470057339.vae043.pub2 may be missing for title: EnvStats:  An R Package for Environmental Statistics

INVALID DOIs

- None
```

**Comment by karthik** [comment]
@flor14 Can you please fix the missing DOIs for the references noted above? After that I will proceed with the next steps to getting your paper accepted.

**Comment by flor14** [comment]
I have just added doi and url to paper.bib. I suppose it solves this

Thank you

Sent from Mail for Windows 10

**Comment by karthik** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- None

MISSING DOIs

- https://doi.org/10.1016/j.ecoenv.2016.08.023 may be missing for title: Aquatic toxicity structure-activity relationships for the zwitterionic surfactant alkyl dimethyl amine oxide to several aquatic species and a resulting species sensitivity distribution
- https://doi.org/10.2903/j.efsa.2013.3290 may be missing for title: Guidance on tiered risk assessment for plant protection products for aquatic organisms in edge-of-field surface waters
- https://doi.org/10.18637/jss.v064.i04 may be missing for title: fitdistrplus: An R Package for Fitting Distributions
- https://doi.org/10.1002/etc.5620190233 may be missing for title: Applying species-sensitivity distributions in ecological risk assessment: assumptions of distribution type and sufficient numbers of species
- https://doi.org/10.1201/9781420032314 may be missing for title: Species sensitivity distributions in ecotoxicology
- https://doi.org/10.1016/s0025-326x(01)00327-7 may be missing for title: Species sensitivity distributions: data and model choice
- https://doi.org/10.1002/9780470057339.vae043.pub2 may be missing for title: EnvStats:  An R Package for Environmental Statistics

INVALID DOIs

- None
```

**Comment by flor14** [comment]
I fixed this again according to the autor’s guidelines https://joss.readthedocs.io/en/latest/submitting.html
Now?

Thank you!

Sent from Mail for Windows 10

**Comment by karthik** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- None

MISSING DOIs

- https://doi.org/10.1016/j.ecoenv.2016.08.023 may be missing for title: Aquatic toxicity structure-activity relationships for the zwitterionic surfactant alkyl dimethyl amine oxide to several aquatic species and a resulting species sensitivity distribution
- https://doi.org/10.2903/j.efsa.2013.3290 may be missing for title: Guidance on tiered risk assessment for plant protection products for aquatic organisms in edge-of-field surface waters
- https://doi.org/10.18637/jss.v064.i04 may be missing for title: fitdistrplus: An R Package for Fitting Distributions
- https://doi.org/10.1002/etc.5620190233 may be missing for title: Applying species-sensitivity distributions in ecological risk assessment: assumptions of distribution type and sufficient numbers of species
- https://doi.org/10.1201/9781420032314 may be missing for title: Species sensitivity distributions in ecotoxicology
- https://doi.org/10.1016/s0025-326x(01)00327-7 may be missing for title: Species sensitivity distributions: data and model choice
- https://doi.org/10.1002/9780470057339.vae043.pub2 may be missing for title: EnvStats:  An R Package for Environmental Statistics

INVALID DOIs

- None
```

**Comment by karthik** [comment]
@flor14 Looks like you removed all the dois?

**Comment by flor14** [comment]
After the failed attempts, I regenerated the file again. I added again the
DOIs now to this new file.

**Comment by karthik** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.1016/j.ecoenv.2016.08.023 is OK
- 10.2903/j.efsa.2013.3290 is OK
- 10.1002/etc.5620190233 is OK
- 10.1201/9781420032314 is OK
- 10.1002/etc.5620190233 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by karthik** [comment]
@flor14 Great! Now can you archive your software on Zenodo and share a DOI here please?

**Comment by flor14** [comment]
Here is the DOI number: 10.5281/zenodo.3233033

**Comment by karthik** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00785/joss.00785/10.21105.joss.00785.pdf)

**Comment by karthik** [comment]
@whedon set 10.5281/zenodo.3233033 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.3233033" target="_blank">10.5281/zenodo.3233033</a> is the archive.

**Comment by karthik** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.1016/j.ecoenv.2016.08.023 is OK
- 10.2903/j.efsa.2013.3290 is OK
- 10.1002/etc.5620190233 is OK
- 10.1201/9781420032314 is OK
- 10.1002/etc.5620190233 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/718

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/718, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by karthik** [comment]
This is ready @openjournals/joss-eics

**Comment by flor14** [comment]
A subindex is missing, also the format of the citation is not correct.
Can I change it or is late?

**Comment by kyleniemeyer** [comment]
@flor14 please go ahead and make any changes to the paper

**Comment by flor14** [comment]
Can you build it again, please?

**Comment by karthik** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00785/joss.00785/10.21105.joss.00785.pdf)

**Comment by kyleniemeyer** [comment]
@flor14 thanks! I will make the final checks on your submission later today.

**Comment by flor14** [comment]
I made a new change. I realize subscripts should be included also as a
formula with $. Hope it works.

**Comment by kyleniemeyer** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00785/joss.00785/10.21105.joss.00785.pdf)

**Comment by kyleniemeyer** [comment]
Hi @flor14, could you add full affiliation information as in a typical article (e.g., department, city, country)?

Also, I made a small PR to fix a few things, if you could accept and merge that: https://github.com/flor14/shinyssd/pull/9

**Comment by flor14** [comment]
I improved what you mentioned. If citations are ok, and subscripts are
correct in the second paragraph: " HC1, HC5 and HC10"  for me it is ready.
Maybe citations could be more detailed in the author's guidelines?
Mentioning the 3 main cases:
1- @author:2001  ->   author et al. (2001)
2- [@author:2001] -> (author et al. 2001)
3- [@author1:2001; @author2:2001]  -> (author1 et al. 2001; author2 et al.
2002)?
I mention this just to save your time.

Thank you Kyle! You was very kind

**Comment by kyleniemeyer** [comment]
@flor14 yes, the citation documentation could be improved... I've had to fix this issue in a number of papers.

**Comment by kyleniemeyer** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #785 with the following error: 

 Error producing PDF.
! Double subscript.
l.316 ...estimators \(HC_1\), \(HC_5\) and \(HC_1_

Looks like we failed to compile the PDF

**Comment by kyleniemeyer** [comment]
@flor14 for the last subscript, I think you need to have `HC_{10}` (or however you need it styled)

**Comment by flor14** [comment]
I've changed subscripts to this now:  $HC_1$, $HC_5$ and $HC_{10}$
The problem seems the last one with 2 digits

**Comment by kyleniemeyer** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00785/joss.00785/10.21105.joss.00785.pdf)

**Comment by kyleniemeyer** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.1016/j.ecoenv.2016.08.023 is OK
- 10.2903/j.efsa.2013.3290 is OK
- 10.1002/etc.5620190233 is OK
- 10.1201/9781420032314 is OK
- 10.1002/etc.5620190233 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/721

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/721, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by kyleniemeyer** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1133851035594891265) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/722
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.00785](https://doi.org/10.21105/joss.00785)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? notify your editorial technical team...

**Comment by kyleniemeyer** [comment]
Congrats @flor14  on your submission's publication in JOSS! Thanks to @kylehamilton for reviewing and @karthik for editing!

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](http://joss.theoj.org/papers/10.21105/joss.00785/status.svg)](https://doi.org/10.21105/joss.00785)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.00785">
  <img src="http://joss.theoj.org/papers/10.21105/joss.00785/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: http://joss.theoj.org/papers/10.21105/joss.00785/status.svg
   :target: https://doi.org/10.21105/joss.00785
```
This is how it will look in your documentation:

[![DOI](http://joss.theoj.org/papers/10.21105/joss.00785/status.svg)](https://doi.org/10.21105/joss.00785)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: http://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.salsalabs.org/donate-to-joss
